app/javascript/react/components/MonsterTile.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { Link } from "react-router-dom"
+
+const MonsterTile = props => {
+  const { monster } = props
```

Comment:
You could also destructure these on line 4
```js
const MonsterTile = ({ monster }) => {
```

Reply:
Done